src/scaict_uwu/libs/services/service_container.py

```
@@ -0,0 +1,87 @@
+"""
+Module for service container.
+"""
+
+# Standard imports
+import sys
+from typing import Callable
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)